MycroftAI/mycroft-core
# Feature/mark 2

## Description
Scaffolding around led activity related to speech activity

## How to test
say hey mycroft - should see a blue chase ring, say what time is it - should see pulsating red led ring

## Contributor license agreement signed?
yes
